contracts/interfaces/IPeriphery.sol

```diff
@@ -73,6 +73,24 @@ interface IPeriphery is CustomErrors {
             int24 _tickAfter
         );
 
+    /// @dev Ensures a fully collateralised VT swap given that a proper value of the variable factor is passed.
+    /// @param params Parameters to be passed to the swap function in the periphery.
+    /// @param variableFactorStartToPresentWad The variable factor between pool's start date and present (in wad).
+    function fullyCollateralisedVTSwap(
+        SwapPeripheryParams memory params,
+        uint256 variableFactorStartToPresentWad
```

Comment:
for consistency with the rest of the codebase can we rename variableFactorStartToPresentWad to variableFactorFromStartToNowWad